### PR TITLE
Add information on minimum .env.local setup

### PR DESCRIPTION
**Pull request by tukantje**
Updates README file to include information on `.env.local` file setup.

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -72,14 +72,25 @@ This will start a server on the `http://localhost:5000/`
...
 The app has some default configuration, but it's highly encouraged to define your own.
 
 ### Local configuration
 
 Make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.
 
+#### What is the mandatory minimum to be able to run the application?
+
+Your environment MUST have the following variables defined:
+`REACT_APP_INFURA_KEY`=YOUR_INFURA_KEY
```
[the remark this answers is not in the document]
Comment (tukantje) [reply README.md:86]: Right now, you'd only have errors in your console whilst not being able to see the swap widget even, due to infura calls failing.

Validating and showing nice errors for env vars could be a nice addition actually.

**Review by alfetopito — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -72,14 +72,25 @@ This will start a server on the `http://localhost:5000/`
...
+`REACT_APP_NETWORK_URL_1`=https://mainnet.infura.io/v3/{YOUR_INFURA_KEY}
+`REACT_APP_NETWORK_URL_5`=https://goerli.infura.io/v3/{YOUR_INFURA_KEY}
```
Comment (anxolin) [inline README.md:88]: so this is also required, is not using infura? also, not Gnosis Chain?

Comment (tukantje) [reply README.md:88]: Will open a follow up PR.

File: README.md [hunk 3]
```diff
@@ -72,14 +72,25 @@ This will start a server on the `http://localhost:5000/`
...
+
+Additionally, to successfully run E2E / Integration tests locally you must have;
+`INTEGRATION_TESTS_INFURA_KEY`=YOUR_INFURA_KEY
+`INTEGRATION_TESTS_PRIVATE_KEY`=YOUR_TEST_WALLET_PRIVATE_KEY
```
[the remark this answers is not in the document]
Comment (tukantje) [reply README.md:92]: Will open a follow up PR.

**Pull request merged**